src/test/java/com/microsoft/sqlserver/jdbc/TestUtils.java

```diff
@@ -63,22 +63,23 @@
     private static ArrayList<SqlType> types = null;
     private static final char[] HEXCHARS = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E',
             'F'};
-    
+
     final static int ENGINE_EDITION_FOR_SQL_AZURE = 5;
     final static int ENGINE_EDITION_FOR_SQL_AZURE_DW = 6;
     final static int ENGINE_EDITION_FOR_SQL_AZURE_MI = 8;
     private static Boolean isAzure = null;
     private static Boolean isAzureDW = null;
     private static Boolean isAzureMI = null;
 
-    /*
-     * SERVERPROPERTY('EngineEdition') can be used to determine whether the db server is SQL Azure.
-     * It should return 6 for SQL Azure DW. This is more reliable than @@version or
-     * serverproperty('edition').
-     * Reference: http://msdn.microsoft.com/en-us/library/ee336261.aspx
+    /**
+     * SERVERPROPERTY('EngineEdition') can be used to determine whether the db server is SQL Azure. It should return 6
+     * for SQL Azure DW. This is more reliable than @@version or serverproperty('edition'). Reference:
+     * http://msdn.microsoft.com/en-us/library/ee336261.aspx
+     * 
+     * SERVERPROPERTY('EngineEdition') means Database Engine edition of the instance of SQL Server installed on the
+     * server.
      * 
-     * SERVERPROPERTY('EngineEdition') means
-     * Database Engine edition of the instance of SQL Server installed on the server.
+     * <pre>
```

Comment:
why do we have pre tags here?

Reply:
as discussed, this is necessary to prefer formatter from making this unreadable